anaconda.spec.in

```diff
@@ -286,9 +286,19 @@ Requires: skopeo
 Requires: nvme-cli
 # Needed for bootc
 Requires: podman
+Requires: bootc
+Requires: bootupd
 # needed for encrypted DNS
 Requires: dnsconfd
 Requires: dnsconfd-dracut
+Requires: coreutils
+Requires: /usr/bin/chcon
+Requires: selinux-policy
```

Comment:
I wonder if there could be some weird environment where SELinux is not avalable yet Anaconda is used ? Maybe some containers or something ? Well, we will find out if people start complaining. :)

BTW, any idea about impact of enabling SELinux on media size ?